astropy/coordinates/builtin_frames/icrs_cirs_transforms.py

```diff
@@ -119,8 +119,11 @@ def cirs_to_cirs(from_coo, to_frame):
 @frame_transform_graph.transform(FunctionTransform, ICRS, GCRS)
 def icrs_to_gcrs(icrs_coo, gcrs_frame):
     # first set up the astrometry context for ICRS<->GCRS
-    pv = np.array([gcrs_frame.obsgeoloc.value,
-                   gcrs_frame.obsgeovel.value])
+    pv = np.array([gcrs_frame.obsgeoloc.xyz.value,
```

Comment:
This is mostly a stylistic question: These 5 lines are repeated 3 times in this file. Maybe it would be worthwile to create a helper function?